tests/dogfood.rs

```diff
@@ -18,20 +18,38 @@ fn dogfood_clippy() {
     }
     let root_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
 
-    let output = Command::new(&*CLIPPY_PATH)
-        .current_dir(root_dir)
-        .env("CLIPPY_DOGFOOD", "1")
-        .env("CARGO_INCREMENTAL", "0")
-        .arg("clippy-preview")
-        .arg("--all-targets")
-        .arg("--all-features")
-        .arg("--")
-        .args(&["-D", "clippy::all"])
-        .args(&["-D", "clippy::internal"])
-        .args(&["-D", "clippy::pedantic"])
-        .arg("-Cdebuginfo=0") // disable debuginfo to generate less data in the target dir
-        .output()
-        .unwrap();
+    let output = if cfg!(feature = "internal-lints") {
```

Comment:
You can save the command in a variable and then reduce the if:

```rust
let mut command = Command::new(..)......;

if cfg!(feature = "internal-lints") {
    command.args(&["-D", "clippy::internal"]);
}

let output = command.output().unwrap();
```

Reply:
Merged the changes into the last commit :+1: